[2022-11-22 03:12:43,406] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.checkfiles.check_energy_file_2 manual__2022-11-22T03:12:40.362784+00:00 [queued]>
[2022-11-22 03:12:43,458] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_solar.checkfiles.check_energy_file_2 manual__2022-11-22T03:12:40.362784+00:00 [queued]>
[2022-11-22 03:12:43,459] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-22 03:12:43,461] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-11-22 03:12:43,462] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-22 03:12:43,539] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): checkfiles.check_energy_file_2> on 2022-11-22 03:12:40.362784+00:00
[2022-11-22 03:12:43,551] {standard_task_runner.py:52} INFO - Started process 162 to run task
[2022-11-22 03:12:43,573] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_solar', 'checkfiles.check_energy_file_2', 'manual__2022-11-22T03:12:40.362784+00:00', '--job-id', '111', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocess.py', '--cfg-path', '/tmp/tmpnvzekbpk', '--error-file', '/tmp/tmp0g0yos_v']
[2022-11-22 03:12:43,576] {standard_task_runner.py:80} INFO - Job 111: Subtask checkfiles.check_energy_file_2
[2022-11-22 03:12:43,884] {task_command.py:369} INFO - Running <TaskInstance: etl_solar.checkfiles.check_energy_file_2 manual__2022-11-22T03:12:40.362784+00:00 [running]> on host 42206527e50b
[2022-11-22 03:12:44,484] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_solar
AIRFLOW_CTX_TASK_ID=checkfiles.check_energy_file_2
AIRFLOW_CTX_EXECUTION_DATE=2022-11-22T03:12:40.362784+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-22T03:12:40.362784+00:00
[2022-11-22 03:12:44,497] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-11-22 03:12:44,507] {subprocess.py:74} INFO - Running command: ['bash', '-c', '/opt/***/utilities/check_file.sh /opt/***/utilities/storage/energy_2.csv']
[2022-11-22 03:12:44,549] {subprocess.py:85} INFO - Output:
[2022-11-22 03:12:44,578] {subprocess.py:92} INFO - File /opt/***/utilities/storage/energy_2.csv does not exist
[2022-11-22 03:12:44,581] {subprocess.py:96} INFO - Command exited with return code 0
[2022-11-22 03:12:44,771] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_solar, task_id=checkfiles.check_energy_file_2, execution_date=20221122T031240, start_date=20221122T031243, end_date=20221122T031244
[2022-11-22 03:12:44,862] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-22 03:12:45,064] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
